src/common/Error/index.js

```diff
@@ -0,0 +1,28 @@
+import { StyledNavLink } from "../Header/Navigation/styled";
```

Comment:
Why are you importing this StyledNavLink from our navigation? you should make a different styling in styled.js, we dont have for example this white border ❗